src/VirtualClient/VirtualClient.Dependencies/CUDAAndNvidiaGPUDriverInstallation.cs

```diff
@@ -38,53 +43,65 @@ public CudaAndNvidiaGPUDriverInstallation(IServiceCollection dependencies, IDict
             this.systemManager = dependencies.GetService<ISystemManagement>();
             this.stateManager = this.systemManager.StateManager;
             this.fileSystem = this.systemManager.FileSystem;
-        }
+            this.packageManager = this.systemManager.PackageManager;
+        }        
 
         /// <summary>
-        /// The version of CUDA to be installed.
+        /// The version of CUDA to be installed in Linux Systems
         /// </summary>
-        public string CudaVersion
+        public string LinuxCudaVersion
         {
             get
             {
-                return this.Parameters.GetValue<string>(nameof(CudaAndNvidiaGPUDriverInstallation.CudaVersion), string.Empty);
+                return this.Parameters.GetValue<string>(nameof(CudaAndNvidiaGPUDriverInstallation.LinuxCudaVersion), string.Empty);
             }
 
             set
             {
-                this.Parameters[nameof(CudaAndNvidiaGPUDriverInstallation.CudaVersion)] = value;
+                this.Parameters[nameof(CudaAndNvidiaGPUDriverInstallation.LinuxCudaVersion)] = value;
             }
         }
 
         /// <summary>
-        /// The version of Nvidia GPU driver to be installed.
+        /// The version of Nvidia GPU driver to be installed in Linux Systems
         /// </summary>
-        public string DriverVersion
+        public string LinuxDriverVersion
         {
             get
             {
-                return this.Parameters.GetValue<string>(nameof(CudaAndNvidiaGPUDriverInstallation.DriverVersion), string.Empty);
+                return this.Parameters.GetValue<string>(nameof(CudaAndNvidiaGPUDriverInstallation.LinuxDriverVersion), string.Empty);
             }
 
             set
             {
-                this.Parameters[nameof(CudaAndNvidiaGPUDriverInstallation.DriverVersion)] = value;
+                this.Parameters[nameof(CudaAndNvidiaGPUDriverInstallation.LinuxDriverVersion)] = value;
             }
         }
 
         /// <summary>
-        /// The local runfile to install Cuda and Nvidia GPU driver.
+        /// The local runfile to install Cuda and Nvidia GPU driver in Linux Systems
         /// </summary>
-        public string LocalRunFile
+        public string LinuxLocalRunFile
         {
             get
             {
-                return this.Parameters.GetValue<string>(nameof(CudaAndNvidiaGPUDriverInstallation.LocalRunFile), string.Empty);
+                return this.Parameters.GetValue<string>(nameof(CudaAndNvidiaGPUDriverInstallation.LinuxLocalRunFile), string.Empty);
             }
 
             set
             {
-                this.Parameters[nameof(CudaAndNvidiaGPUDriverInstallation.LocalRunFile)] = value;
+                this.Parameters[nameof(CudaAndNvidiaGPUDriverInstallation.LinuxLocalRunFile)] = value;
+            }
+        }
+
+        /// <summary>
+        /// Determines whether Reboot is required or not after Driver installation
```

Comment:
This now only applies to Windows, if Windows never needs a reboot, let's just remove this parameter. When appearing on the profile this will confuse user

Reply:
resolving as per discussion